internal/pkg/skuba/kubernetes/kubelet_test.go

```diff
@@ -18,40 +18,91 @@
 package kubernetes
 
 import (
+	"crypto/sha1"
+	"fmt"
+	"os"
 	"testing"
 
-	v1 "k8s.io/api/core/v1"
+	batchv1 "k8s.io/api/batch/v1"
+	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/client-go/kubernetes/fake"
+	ktest "k8s.io/client-go/testing"
+
+	"github.com/SUSE/skuba/pkg/skuba"
 )
 
+func TestGenerateKubeletRootCert(t *testing.T) {
+	t.Run("generate kubelet root cert", func(t *testing.T) {
+		if err := GenerateKubeletRootCert(); err != nil {
+			t.Errorf("error not expected, but error reported generating pki: %v", err)
+		}
+
+		if err := GenerateKubeletRootCert(); err != nil {
+			t.Errorf("error not expected, but error reported when pki exist: %v", err)
+		}
```

Comment:
intended? or duplicated?

Reply:
intended. this is to test `generate kubelet root cert when already exist`.

Reply:
added comment in-code.